### PR TITLE
Enables bot to hear to more than one page

### PR DESCRIPTION
**Pull request by andremw**
#5 
This changes enables one single instance of the plugin to talk to more than one page.
There are some things pending: 

- [ ] Unit tests (I haven't managed to make "coveralls" create the file correctly yet, so I can't run the tests.)
- [ ] Handle optional parameters
- [ ] Update docs

### CODE REVIEW

File: lib/messenger_bot.js [hunk 1]
```diff
@@ -557,18 +554,27 @@ class MessengerBot extends BaseBot {
...
       uri: `${baseURL}/${userId}`,
-      qs: { access_token: this.credentials.pageToken },
+      qs: { access_token: this.__getAccessToken(requestOptions.sendAsPageId) },
       json: true,
     };
 
     return request(options);
   }
 
+  __getAccessToken(pageId = this.__associatedUpdate.recipient.id) {
```
Comment (jdwuarin) [inline lib/messenger_bot.js:570]: You won't be able to use default params if you want your tests to pass on node 4 and 5

**Comment by jdwuarin** [comment]
Looks like good points to keep in mind. I'll publish the new version of botmaster once this works then. I should add how to run tests with your own credentials in the Readme. I'll have a go at doing this now

**Comment by Rikhart** [comment]
Hi is this issue still open?, when will be part of the core?.

Any news, thanks for your answers.

**Comment by andremw** [comment]
Hey, @Rikhart. For some reasons that I didn't expect I haven't found time to invest on this, but it's still important for me as well. I will get back to it as soon as possible.

**Comment by bobber205** [comment]
Any updates on this? I will need to use this functionality soon.

**Comment by andremw** [comment]
Sorry, guys, but I recently left the company where I was using Botmaster to start my own business, so I won't be able to work on this in the foreseeable future.

@bobber205 @jdwuarin @Rikhart

**Comment by jdwuarin** [comment]
Hey - good to hear you're trying it out on your own!
Pretty stomped right now @bobber205 . But have been wanting to add this feature for a while. So will try and do this over the weekend once I've assessed how long it should take me (shouldn't be that long though)

**Comment by bobber205** [comment]
Thanks so much @jdwuarin !

**Comment by jdwuarin** [comment]
Fixed by: 477650a77d18a5bf104562bdb66b734f57bd97e3

**Pull request closed without merging**